File: shanny-blog-java/blog-pojo/src/main/java/com/shanny/entity/About.java
package com.shanny.entity;

import com.shanny.enums.AboutEnum;
import com.shanny.enums.UserEnum;
import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import lombok.Data;
import org.springframework.format.annotation.DateTimeFormat;

import java.time.LocalDateTime;

@Data
@Entity
@Table(name = "abouts")
public class About {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @NotNull
    private Boolean isActive;

    @Column(length = 30, nullable = false)
    @NotNull
    private String name;

    @Column(length = 300, nullable = false)
    @NotNull
    private String introduce;

    @Column(length = 500, nullable = false)
    @NotNull
    private String avatar;

    @Column(length = 200, nullable = false)
    @NotNull
    private String tag;

    @Column(length = 200)
    private String github;

    @Column(length = 200)
    private String steam;

    @Column(length = 200)
    private String web;

    @Column(length = 200)
    private String biliBili;

    @Column(length = 200)
    private String other;

    @Column(nullable = false)
    @NotNull
    @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime createTime;

    @Column(nullable = false)
    @NotNull
    @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime updateTime;
}

File: shanny-blog-net/blog-pojo/Entities/About.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace blog_pojo.Entities
{
    [Table("abouts")]
    public class About
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Column("id")]
        public long Id { get; set; }

        [Required]
        [Column("isActive")]
        public bool IsActive { get; set; }

        [MaxLength(30)]
        [Required]
        [Column("name")]
        public string Name { get; set; } = string.Empty;

        [MaxLength(300)]
        [Required]
        [Column("introduce")]
        public string Introduce { get; set; } = string.Empty;

        [MaxLength(500)]
        [Required]
        [Column("avatar")]
        public string Avatar { get; set; } = string.Empty;

        [MaxLength(200)]
        [Required]
        [Column("tag")]
        public string Tag { get; set; } = string.Empty;

        [MaxLength(200)]
        [Column("github")]
        public string Github { get; set; } = string.Empty;

        [MaxLength(200)]
        [Column("steam")]
        public string Steam { get; set; } = string.Empty;

        [MaxLength(200)]
        [Column("web")]
        public string Web { get; set; } = string.Empty;

        [MaxLength(200)]
        [Column("biliBili")]
        public string BiliBili { get; set; } = string.Empty;

        [MaxLength(200)]
        [Column("other")]
        public string Other { get; set; } = string.Empty;

        [Required]
        [Column("createTime")]
        public DateTime CreateTime { get; set; }

        [Required]
        [Column("updateTime")]
        public DateTime UpdateTime { get; set; }
    }
}
